### PR TITLE
expose label color

### PR DESCRIPTION
**Pull request by alecjacobson**
Can change base:dev after templates-and-overloads is merged into dev.

#### Check all that apply (change to `[x]`)
- [x] All changes meet [libigl style-guidelines](https://libigl.github.io/style-guidelines/).
- [ ] Adds new .cpp file.
- [ ] Adds corresponding unit test.
- [ ] Adds corresponding python binding.
- [x] This is a minor change.

### CODE REVIEW

File: include/igl/opengl/glfw/imgui/ImGuiMenu.cpp [hunk 1]
```diff
@@ -113,7 +113,9 @@ IGL_INLINE bool ImGuiMenu::mouse_down(int button, int modifier)
 
 IGL_INLINE bool ImGuiMenu::mouse_up(int button, int modifier)
 {
-  return ImGui::GetIO().WantCaptureMouse;
+  //return ImGui::GetIO().WantCaptureMouse;
+  // !! Should not steal mouse up
```
Comment (jdumas) [inline include/igl/opengl/glfw/imgui/ImGuiMenu.cpp]: Why this change?

Comment (alecjacobson) [reply include/igl/opengl/glfw/imgui/ImGuiMenu.cpp]: This is just general GUI good manners. Don't steal mouse up. Everyone should be allowed to know if mouse up happened.

Comment (jdumas) [reply include/igl/opengl/glfw/imgui/ImGuiMenu.cpp]: I'm not sure about that. Why is that different from a mouse down event? This flag means that mouse event happens on a imgui widget, so it should not be dispatched/may be hidden from the rest of the application. But maybe you have a specific example of interaction where this was problematic?

**Comment by jdumas** [comment]
Is there any reason you're in such a rush to merge those changes into `dev`? I'm still not sure about whether `ImGui::GetIO().WantCaptureMouse` should be commented out...
